apps/wakunode2/config.nim

```diff
@@ -201,6 +201,11 @@ type
       defaultValue: ""
       name: "rln-relay-cred-password" }: string
 
+    rlnRelayTreePath* {.
+      desc: "Path to the RLN merkle tree db",
```

Comment:
What type of db is expected? Maybe it is worth specifying it a bit more :)

Reply:
Yes! good idea, addressed in 69dc716e